### DISCUSSION TITLE
Qual seria a associação correta no curso de Acesso à dados

### DISCUSSION THREAD

**Discussion by rafaelmoraism1** [balta-io/forum#310] [Tira Dúvidas] [answered]
Olá, estou finalizando o curso de Acesso à Dados e estou com uma dúvida sobre Qual seria a associação correta entre as classes?
As classes propostas no curso são: Category, user, role, post e tag. Mas preciso realizar a associação entre elas. 
Exemplo: Uma categoria possui usuário. Um role possui oque? Um post possui ou é possuído? 
Preciso esclarecer essas dúvidas para terminar o projeto.
Fico no aguardo!
Att, Rafael

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rafaelmoraism1, como vai?

Exemplo: Uma categoria possui usuário. Um role possui oque? Um post possui ou é possuído?
- Um post pertence a uma categoria
- Uma categoria possui vários posts
- Um usuário pertence a um role
- Um role possui muitos usuários
- Um post possui muitas tags
- Uma tag possui muitos posts

Seria isto?

**Reply by rafaelmoraism1** [reply]
Bom dia, Balta.
Seria isso mesmo. Depois que realizei a pergunta aqui, voltei nos vídeos iniciais do curso, onde foi construído o DB e lá é explicado a relação entre as classes. Devo ter esquecido de anotar quando vi a aula.
Muito obrigado pela atenção.

**Reply by andrebaltieri** [reply]
Tranquilo!!! 🚀
